string mission_id
string task
float64 delay
---
bool done